testing/run_unit_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+MINCOVERAGE=92
+SCRIPTPATH=$(readlink -f "$0")
+TESTDIR=`dirname $SCRIPTPATH`
+BASEDIR=`readlink -f $TESTDIR/..`
+cd $BASEDIR || exit 1
```

Comment:
all the || exit 1 start to get ugly... I'd just add -e to the top and then explicitly allow whatever commands are allowed to fail.

Reply:
Addressed.

Reply:
Except for the "exit 1"?